modules/dag-l1/src/main/scala/org/tessellation/dag/l1/cli/method.scala

```diff
@@ -62,6 +62,8 @@ object method {
 
     val l0SeedlistPath = None
 
+    val prioritySeedlistPath = None
+    val realignWithNonPrioritySeedlist = false
```

Comment:
also, what happens when there are different peers on the seedlist than on the priority seedlist?
I'd still opt for a flag in the original seedlist file, to mark some peers as priority (cc: @ryle-goehausen)

Reply:
I'll defer to @ryle-goehausen for the peers on the seedlists question.

For the other question, the flag dictates which seedlist to use for the "majority".
Regardless which is passed in, if there are no peers in the set then the original non-majority checking code is used.
We can change that to raise an error if that is more appropriate.
Originally there were not new methods so I was supporting both cases since the same service was used across modules. @TheMMaciek suggested new methods instead.

Reply:
@sdavidp I meant to have a flag next to the peer within the original seedlist csv, like:
```
peerId,node_alias,trust,...,isPriorityPeer
```

Reply:
> Regardless which is passed in, if there are no peers in the set then the original non-majority checking code is used.
is it correct behavior? if we're adding majority selection, then IMO it should be always working, not selectively

Reply:
I believe the current implementation will use the majority picking version if there are priority peers. We can't do majority picking without it. Since passing priority peers is optional we only use that algorithm if they're present. If they are present then any issue during majority selection raises an error.

Reply:
If we change the schema for the seed list we need to ensure parsing support for nodectl for this and the future, and it doesn't make sense to add a column like this for a feature that won't be supported forever. It'd still rather just have it embedded separately to contain the additional information. In general if we're talking about the problem of applying labels in order of potentially different types, (specifically in the future any nested forms of labels,) they in general may not present easily as CSV data, meaning we'd want to manage multiple files in the future for different options.

Reply:
> we need to ensure parsing support for nodectl for this

you need support of nodectl for a priority seedlist file as well

Reply:
Summary of discussion: 

We'll rely on user provided env for initial GL0 peer to populate initial discovery and alignment check / discovery of snapshot signature peers to handle updating of peers list. We'll assume priority list matches user supplied for discovery purposes. We don't need to add 3rd CLI parameter for GL0 seed list to distinguish it. We don't need to modify the schema of the existing seed list -- because we're NOT passing in the GL0 seed list to the other places where it'll be used (as this is not being used in core,) so even if we modified the schema we'd be unable to use it since we don't have access to it (it's a None.) So we need to use priority seed list and keep type as embedded.